File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/inventory/services/ReservationService.java
package com.metaengine.demo.inventory.services;

import com.metaengine.demo.inventory.aggregates.StockItem;
import java.util.List;

/** Reservation service for the StockItem aggregate. */
public class ReservationService {

    /** Reserve a new StockItem from a partial input. */
    public StockItem create(StockItem input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a reserved StockItem by id; returns null if not found. */
    public StockItem findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List reserved StockItems, capped at the supplied limit. */
    public List<StockItem> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Release a reservation by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-004/a-mcp/output/inventory/services/reservation_service.py
from typing import Optional
from inventory.aggregates.stock_item import StockItem

"""ReservationService service."""
class ReservationService:
    def create(self, input: StockItem) -> StockItem:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[StockItem]:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[StockItem]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

